ocaml/idl/gen_lifecycle.ml

```diff
@@ -15,7 +15,7 @@
 open Datamodel_types
 
 let current_version =
-  Scanf.sscanf Xapi_version.version "%s@-%s" (fun v _rest -> v)
+  Xapi_version.(Printf.sprintf "%d.%d" xapi_version_major xapi_version_minor)
```

Comment:
We'll lose the micro version here: the old version of the function kept everything before the `-`.

Reply:
That commit has been dropped